pcweb/components/webpage/navbar/navbar.py

```diff
@@ -37,88 +37,88 @@ def resources_item(text, url, icon):
-                    resources_item(
-                        "Github Discussions",
-                        "https://github.com/orgs/reflex-dev/discussions",
-                        "message-circle-question",
-                    ),
-                    resources_item(
-                        "Contribute to Reflex",
-                        constants.CONTRIBUTING_URL,
-                        "file-json-2",
-                    ),
-                    resources_item("Changelog", changelog.path, "list-checks"),
-                    direction="column",
-                    align_items="start",
-                    padding_left="20px",
-                    padding_y="20px",
-                    spacing="2",
-                ),
-                rx.flex(
-                    resource_header("Resources"),
-                    resources_item(
-                        "Component Library", library.path, "layout-panel-left"
-                    ),
-                    resources_item(
-                        "Roadmap",
-                        constants.ROADMAP_URL,
-                        "map-pinned",
-                    ),
-                    resources_item("Blog", blogs.path, "text"),
-                    resources_item("FAQ", faq.path, "list-todo"),
-                    resources_item("Common Errors", errors.path, "octagon-alert"),
-                    direction="column",
-                    align_items="start",
-                    padding_top="20px",
-                    padding_bottom="20px",
-                    spacing="2",
-                ),
-                rx.flex(
-                    resource_header("Hosting"),
-                    resources_item(
-                        "Deployment", hosting.deploy_quick_start.path, "globe"
-                    ),
-                    resources_item("Self-Hosting", hosting.self_hosting.path, "server"),
-                    direction="column",
-                    background="linear-gradient(218deg, #1D1B23 -35.66%, #131217 100.84%);",
-                    border_left="1px solid rgba(29, 29, 32, 0.08);",
-                    align_items="start",
-                    height="230px",
-                    padding_top="20px",
-                    padding_left="20px",
-                    padding_bottom="20px",
-                    padding_right="20px",
-                    spacing="2",
-                ),
-                spacing="6",
-            ),
-            border="1px solid rgba(29, 29, 32, 0.08);",
-            background="linear-gradient(180deg, rgba(29, 27, 33, 0.95) 0%, rgba(20, 19, 24, 0.95) 100%);",
-            box_shadow="0px 24px 54px -17px rgba(13, 12, 16, 0.30), 0px 0px 0px 1px rgba(93, 93, 107, 0.29), 0px 0px 64px 5px rgba(53, 51, 60, 0.30) inset;",
-            max_width="1000px",
-            height="230px",
-            padding="0",
-            overflow="hidden",
-        ),
-    )
+# def resources__section(style):
```

Comment:
Is this intentional?

Reply:
yes as this code was not being used anywhere and was duplicate code